src/components/portrait/portrait-test.stories.tsx

```diff
@@ -34,29 +32,21 @@ export default {
         type: "select",
       },
     },
-    altSpecialCharacters: specialCharacters,
   },
 } as ComponentMeta<typeof Portrait>;
 
 export const Default = ({
   alt,
-  altSpecialCharacters,
   ...args
 }: {
   alt?: string;
-  altSpecialCharacters?: string;
 } & PortraitProps) => (
```

Comment:
```suggestion
}: PortraitProps) => (
```
The `alt` prop is now part of the `PortraitProps` interface, so we can simplify this like so